Stores/FeedStore.js

```diff
@@ -10,11 +10,11 @@ var _feeds = [];
 var store = createStore({
 
   setState(feeds) {
-    _feeds = feeds || [];
+    _feeds = (feeds && feeds.slice()) || [];
   },
 
   getState() {
-    return _feeds;
+    return _feeds.slice();
```

Comment:
defensive copying here since we're not using immutable data, just pass a copy to consumers